Covid19Radar/Covid19Radar/ViewModels/Settings/DebugPageViewModel.cs

```diff
@@ -0,0 +1,160 @@
+                case Device.iOS:
+                    os = "iOS";
+                    break;
+                default:
+                    os = "unknown";
+                    break;
+            }
+#if DEBUG
+            os += ",DEBUG";
+#endif
+#if USE_MOCK
+            os += ",USE_MOCK";
+#endif
+
+            // debug info for ./SplashPageViewModel.cs
+            string agree;
+            if (termsUpdateService.IsAllAgreed())
+            {
+                agree = "exists";// (mainly) navigate from SplashPage to HomePage
+                var termsUpdateInfo = await termsUpdateService.GetTermsUpdateInfo();
+                if (termsUpdateService.IsReAgree(TermsType.TermsOfService, termsUpdateInfo))
+                {
+                    agree += "-TermsOfService";
+                }
+                else if (termsUpdateService.IsReAgree(TermsType.PrivacyPolicy, termsUpdateInfo))
+                {
+                    agree += "-PrivacyPolicy";
+                }
+            }
+            else
+            {
+                agree = "not exists"; // navigate from SplashPage to TutorialPage1
+            }
+
+            var ticks = exposureNotificationService.GetLastProcessTekTimestamp(AppSettings.Instance.SupportedRegions[0]);
+            var dt = DateTimeOffset.FromUnixTimeMilliseconds(ticks).ToOffset(new TimeSpan(9, 0, 0));
+            //please check : offset is correct or not
+            //cf: ../../../Covid19Radar.Android/Services/Logs/LogPeriodicDeleteServiceAndroid.cs
+            var lastProcessTekTimestamp = dt.ToLocalTime().ToString("F");
+
+            var exposureNotificationStatus = await Xamarin.ExposureNotifications.ExposureNotification.IsEnabledAsync();
+            var exposureNotificationMessage = await exposureNotificationService.UpdateStatusMessageAsync();
+            // ../../settings.json
+            var str = new[] { "Build: " + os, "Ver: " + AppSettings.Instance.AppVersion,
+                "Region: " + string.Join(",", AppSettings.Instance.SupportedRegions), "CdnUrl: " + AppSettings.Instance.CdnUrlBase,
+                "ApiUrl: " + AppSettings.Instance.ApiUrlBase, "Agree: " + agree, "StartDate: " + userDataService.GetStartDate().ToLocalTime().ToString("F"),
+                "DaysOfUse: " + userDataService.GetDaysOfUse(), "ExposureCount: " + exposureNotificationService.GetExposureCountToDisplay(),
+                "LastProcessTek: " + lastProcessTekTimestamp, " (long): " + ticks, "ENstatus: " + exposureNotificationStatus,
+                "ENmessage: " + exposureNotificationMessage, "Now: " + DateTime.Now.ToLocalTime().ToString("F"), ex};
+            DebugInfo = string.Join(Environment.NewLine, str);
+        }
+        public DebugPageViewModel(INavigationService navigationService, IUserDataService userDataService, ITermsUpdateService termsUpdateService, IExposureNotificationService exposureNotificationService) : base(navigationService)
+        {
+            Title = "Title:DebugPage";
```

Comment:
```suggestion
            Title = "Title:Debug";
```

Debug Pageでも良いかと思いましたが、開発者しか使わないことを考えると、よりシンプルに`Debug`でいいかと思います。